src/models/fire-model.ts

```diff
@@ -30,6 +31,12 @@ const FuelConstants: {[key in LandType]: Fuel} = {
   }
 };
 
+const dist = (c1: Cell, c2: Cell) => {
```

Comment:
When runtime performance becomes important, an easy optimization is to compare see if either xDiff or yDiff are zero -- and just return the difference of the "other" diff -- this avoids the more expensive call to sqrt. If the nearest neighbors are lose in, and thus, more likely to be vertically or horizontally aligned, this will have more impact. If the nearest neighbors is a larger sample, it will have less impact.

something like:

if (xDiff == 0 || yDiff == 0) {
  return Math.abs(xDiff + yDiff);
} else {
  return Math.sqrt...

Or some such. Prolly won't matter much since the number of calcs grow at least as fast as the square of the size of simulation grid. But it's easy.

Reply:
Thanks, fixed now.